tests/DriverManagerTest.php

```diff
@@ -163,6 +163,53 @@ public function testDatabaseUrlPrimaryReplica(): void
         );
     }
 
+    public function testDatabaseServerVersionFromPrimaryReplica(): void
+    {
+        $options = [
+            'driver' => 'pdo_mysql',
+            'primary' => ['url' => 'mysql://foo:bar@localhost:11211/baz?serverVersion=8.0'],
```

Comment:
The `url` parameter is deprecated. Please don't use it unless it's the only way to reproduce the issue. But as far as I understand the issue, it's not related to URL parsing, is it?

Reply:
Okay I didn't know the `url` parameter was deprecated. All the latest Symfony docs still say that is how you are meant to setup the database config.